.github/workflows/integration.yml

```diff
@@ -7,180 +9,243 @@ on:
       - master
   workflow_dispatch: {}
 
-name: integration
-env:
-  CI: true
```

Comment:
I took inspiration from [this thread](https://stackoverflow.com/questions/60006265/whats-the-purpose-of-the-environment-variable-ci-set-to-true-within-a-github-wo), and since it is not used anywhere in tests or other scripts, I took the liberty to remove it.